src/main/java/com/pingcap/tikv/meta/TiSelectRequest.java

```diff
@@ -88,6 +91,14 @@ public SelectRequest buildScan(boolean idxScan) {
     }
   }
 
+  public void skipAggregatesAt(int ordinal) {
```

Comment:
didn't see any usage of this function. Why add this? I got the method's meaning but don't know what does `ordinal` mean.

Reply:
Used in tispark. Ordinal means position